### PR TITLE
Update to .net 6 and add Swagger Filter

### PR DESCRIPTION
**Pull request by kennethlong**
I updated to .Net 6, added a Swagger filter to make Pagination and Sort work with Swagger page, updated nuget package version, updated documentation, updated packages to latest released versions that support .net 6.

### CODE REVIEW

File: src/JHipsterNet.Web/Pagination/Swagger/PageableModelFilter.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+            if (description.ParameterDescriptions.Any(elt => elt.Type == typeof(IPageable))) {
+                operation?.RequestBody?.Content?.Clear();
+            }
+
+            if (operation == null) {
+                return;
+            }
+
+            operation.Parameters.Add(new OpenApiParameter {
```
Comment (nicolas63) [inline src/JHipsterNet.Web/Pagination/Swagger/PageableModelFilter.cs:30]: Maybe we can add a little description in the parameter

**Comment by nicolas63** [comment]
Can you sign the CLA and also update the .tools-versions ?

**Comment by kennethlong** [comment]
I updated .tools-versions in the first commit, let me know if I need to change that.
I committed this from my work machine which has my work address, I added that work address to my account, but CLA is still saying there is a problem.  Let me check everything again.

**Comment by kennethlong** [comment]
It just took like 10 minutes for everything to sync up, looks good now!

**Comment by nicolas63** [comment]
it's ok for me

**Pull request merged**